beacon_node/beacon_chain/src/fetch_blobs.rs

```diff
@@ -0,0 +1,215 @@
+use crate::observed_data_sidecars::ObservableDataSidecar;
```

Comment:
Add module docs about this strategy or link to an external resource explaining it

Reply:
Updated in https://github.com/sigp/lighthouse/pull/6268/commits/dbf4bb8bfdbb3ca37d82b948958ef48670aa0882